cloudkittyclient/common/utils.py

```diff
@@ -281,6 +281,7 @@ class color:
                         'dict_floating': 'Floating IP Charges',
                         'dict_outbound': 'Outbound Charges for Instance',
                         'dict_cloud_storage': 'cloud storage charges(Swift)',
+                        'dict_instance_addon': 'instance addon charges',
```

Comment:
Again, ensure capitalization matches the others:

Cloud Storage Charges (swift)
Instance Add-on Charges